### PR TITLE
Network screen (Remote configs)

### PR DESCRIPTION
**Pull request by AleksKSoftware**
For the network screen was added remote configs for the colours and styles. 
It was created a json for the screen on the Firebase console.

### CODE REVIEW

File: lib/widgets/navigation_drawer.dart [hunk 1]
```diff
@@ -64,23 +70,25 @@ class NavigationDrawer extends StatelessWidget {
...
+      const Padding(padding: EdgeInsets.only(top: 16)),
     ];
     _drawerGroupedItems.forEach((groupItems) {
       children.addAll(_createDrawerGroupWidgets(
         groupItems,
         context,
+        colorScheme,
         _drawerGroupedItems.indexOf(groupItems),
-        withDivider: children.length > 0 && groupItems.withDivider,
+        withDivider: children.isNotEmpty && groupItems.withDivider,
```
Comment (cvicenti) [inline lib/widgets/navigation_drawer.dart]: can this still be 0, though?

File: lib/routes/home/payment_item.dart [hunk 2]
```diff
@@ -177,7 +188,9 @@ class PaymentItem extends StatelessWidget {
       _hideBalance
           ? texts.wallet_dashboard_payment_item_balance_hide
           : texts.wallet_dashboard_payment_item_balance_fee(feeFormatted),
-      style: themeData.accentTextTheme.caption,
+      style: themeData.accentTextTheme.caption.copyWith(
```
Comment (cvicenti) [inline lib/routes/home/payment_item.dart]: What does this do?

File: lib/routes/splash_page.dart [hunk 3]
```diff
@@ -46,9 +48,19 @@ class SplashPageState extends State<SplashPage> {
...
         body: Stack(children: <Widget>[
-      Center(
-        child: SizedBox(),
-      ),
-    ]));
+          Center(
+            child: SizedBox(),
+          ),
+        ]));
+    // return   FutureBuilder<FirebaseRemoteConfig>(
```
Comment (cvicenti) [inline lib/routes/splash_page.dart]: did you mean to leave this here?

Comment (cvicenti) [reply lib/routes/splash_page.dart]: also a single line on lib/routes/home/qr_action_button.dart#54 and a few blocks on lib/services/remote_configs/remote_config_service.dart

File: lib/widgets/static_loader.dart [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
+    return Container();
+  }
+
+  Future<FirebaseRemoteConfig> setupRemoteConfig() async {
+    final FirebaseRemoteConfig remoteConfig = FirebaseRemoteConfig.instance;
+    await remoteConfig.setConfigSettings(RemoteConfigSettings(
+      fetchTimeout: const Duration(seconds: 10),
+      minimumFetchInterval: const Duration(seconds: 10),
+    ));
+    await  remoteConfig.fetchAndActivate();
```
Comment (cvicenti) [inline lib/widgets/static_loader.dart]: nit: double space after 'await'

**Review by cvicenti — DISMISSED** [review]

File: lib/routes/home/payment_item.dart (hunk 2 above)
Comment (AleksKSoftware) [reply lib/routes/home/payment_item.dart]: It copies the object style with one different - colour is changed.

File: lib/widgets/navigation_drawer.dart (hunk 1 above)
Comment (AleksKSoftware) [reply lib/widgets/navigation_drawer.dart]: No, the logic is next drawing divider except of the last item.

File: lib/routes/splash_page.dart (hunk 3 above)
Comment (AleksKSoftware) [reply lib/routes/splash_page.dart]: The commented code was removed.

**Review by RodriFS — APPROVED** [review]

File: lib/widgets/static_loader.dart (hunk 4 above)
Comment (AleksKSoftware) [reply lib/widgets/static_loader.dart]: Refactored

**Pull request merged**